exporters/otlp/trace/src/main/java/io/opentelemetry/exporter/otlp/trace/OtlpGrpcSpanExporterBuilder.java

```diff
@@ -80,7 +80,11 @@ public OtlpGrpcSpanExporterBuilder setTimeout(Duration timeout) {
    */
   public OtlpGrpcSpanExporterBuilder setEndpoint(String endpoint) {
     requireNonNull(endpoint, "endpoint");
-    final URI uri;
+    if (!endpoint.matches("^[a-zA-Z]+:\\/\\/.*")) {
```

Comment:
How about just `endpoint.contains("://")`?  That way we won't miss restoring invalid URL check when we remove support for schemaless URLs.

Reply:
Seems fine. I'll update

Reply:
done